docs/services/matrix.md

```diff
@@ -85,5 +85,29 @@ By default, a SQL-Lite database is used, so we still need to comment it out.
 
 ```
 
-Now we are finish and we can start the container with `docker-compose up -d`
+Now you are finish and you can start the container with `docker-compose up -d`
 
+### Reset password of user 
+
+If you want to reset the assword run 
+```yaml 
+  docker exec -it -u www-data matrix hash_password -p PASSWORD
+```
+
+After the command is done you will become a password hash as std-out. 
+
+After you have generated the password hash you can update the value in the database. First you shoud start a shell in the postgress container with 
+``` yaml
+docker exec -it matrix_postgresql /bin/bash
+```
+Next, you can connect to the database with the command 
+```yaml
+psql -u postgres -d synapse -w S3cr3T
+```
```

Comment:
Use docker-compose to do this:
```bash
docker-compose exec /bin/bash
```
In fact, let's just open up the psql shell, instead of bin/bash, we can also use the environment variables:
```bash
docker-compose exec /bin/sh -c "psql -u postgres -d synapse -w ${POSTGRES_PASSWORD}"
```